pkg/minikube/node/cache.go

```diff
@@ -272,10 +274,7 @@ func imagesInConfigFile() ([]string, error) {
 		return nil, errors.Wrap(err, "read")
 	}
 	if values, ok := configFile[cacheImageConfigKey]; ok {
-		var images []string
-		for key := range values.(map[string]interface{}) {
-			images = append(images, key)
-		}
+		images := slices.Collect(maps.Keys(values.(map[string]interface{})))
```

Comment:
This is little bit too dense. It will help to mote the type assertion out:

```go
m := values.(map[string]interface{})
images := slices.Collect(maps.Keys(m))
```

It will help also to remove the type assertion later or add a comment why it is needed.